configure.ac

```
@@ -263,6 +263,17 @@ fi
 # Checks for OpenCL backend
 AX_OPENCL
 
+# Set the NEKO_BCKND variable
+if test "x${have_cuda}" = xyes; then
+   AC_SUBST(NEKO_BCKND, "CUDA")
+elif test "x${have_hip}" = xyes; then
+   AC_SUBST(NEKO_BCKND, "HIP")
+elif test "x${have_opencl}" = xyes; then
+   AC_SUBST(NEKO_BCKND, "OpenCL")
+else
+   AC_SUBST(NEKO_BCKND, "none")
+fi
```

```suggestion
elif test "x${have_opencl}" = xyes; then
   AC_SUBST(NEKO_BCKND, "OpenCL")
elif test "x${is_sx}" = x1; then
   AC_SUBST(NEKO_BCKND, "SX-Aurora")
else
   AC_SUBST(NEKO_BCKND, "CPU")
fi
```

Suggestions to make it more consistent with the output from Neko_init